apiserver/facades/client/application/application.go

```diff
@@ -2114,39 +2128,42 @@ func (api *APIBase) ApplicationsInfo(ctx context.Context, in params.Entities) (p
 
 		appLife, err := api.applicationService.GetApplicationLife(ctx, tag.Name)
 		if errors.Is(err, applicationerrors.ApplicationNotFound) {
-			err = errors.NotFoundf("application %q", tag.Name)
-		}
-		if err != nil {
-			out[i].Error = apiservererrors.ServerError(err)
+			out[i].Error = apiservererrors.ServerError(errors.NotFoundf("application %s", tag.Name))
```

Comment:
You can do this now as well FYI. Not saying it needs to be done just as an fyi. Might make this hobbling together several things easier.

```
apiservererrors.ParamsErrorf(params.CodeNotFound, "application %q does not exist", tag.Name)
```

Reply:
Nice! resolved